stage1/init/init.go

```diff
@@ -577,16 +577,17 @@ func stage1() int {
 		return 1
 	}
 
-	// create a separate mount namespace so the cgroup filesystems
-	// are unmounted when exiting the pod
+	// Creates a separate mount namespace so the cgroup filesystems
```

Comment:
I think it's important to adhere the sentence formatting, otherwise people use different formats. 

I think the inline documentation should be similar to the godoc's, but I can go either way that's not an issue.